rasa/cli/test.py

```diff
@@ -115,29 +116,51 @@ def run_core_test(args: argparse.Namespace) -> None:
     )
 
 
-def run_nlu_test(args: argparse.Namespace) -> None:
-    """Run NLU tests."""
+async def run_nlu_test_async(
+    config: Optional[Union[Text, Dict]],
```

Comment:
according to your docstring it should be this, no?
```suggestion
    config: Optional[Union[Text, List[Text]],
```

Reply:
True! I changed it as you suggested.